code/game/objects/structures/vehicles/gigadrill.dm

```diff
@@ -7,6 +7,7 @@
 
 /obj/structure/bed/chair/vehicle/gigadrill
 	name = "gigadrill"
+	desc = "An old and dusty vehicle with a mining drill - structure composed of titanium-carbide alloy, with tip and drill lines edged in an alloy of diamond and plasma"
```

Comment:
Missing period at end


Also here's a quote for you

If there's a wall in our way then we smash it down! If there isn't a path, then we carve one ourselves!

Reply:
Excellent, I'll update this tomorrow

Reply:
Implemented